std/format/read.d

```diff
@@ -91,7 +147,7 @@ module std.format.read;
     assert(str.unformatValue!dchar(spec) == '✓');
 }
 
-/// Arrays and static arrays
+/// Strings and arrays can be read as a single value and as a compound type.
```

Comment:
ditto + Strings are actually arrays behind the scenes so I'm not sure this extra mention is necessary.

Reply:
They have got different implementations and beginners might not be aware of strings being essentially the same as arrays.

Reply:
OK, still I think it's better if we put the "String" comment right above the example.